client/test/unit/specs/ExactMatchesRequest.spec.js

```diff
@@ -40,7 +40,7 @@ describe('store > checkManualExactMatches', () => {
     })
 
     it('escape special characters', ()=>{
-        store.dispatch('checkManualExactMatches', '/?.><\\\'\":;\\|][}{=+_-)(*&^%$#@!~`ATHENAE/?.><\\\'\":;\\|][}{=+_-)(*&^%$#@!~`UM 139 LTD./?.><\\\'":;\\|][}{=+_-)(*&^%$#@!~`')
+        store.dispatch('checkManualExactMatches', '/?.><\\\'\":;\\|][}{=+_-)(*&^%$#@!~`ATHENAE/?.><\\\'\":;\\|][}{=+_-)(*&^%$#@!~`UM 139 LTD./?.><\\\'":;\\|][}{=+_-)(*&^%$#@!~` +++++ ------')
```

Comment:
This test fails - when I run it, the actual results now have two spaces (%20) added to the end due to the spaces you've added in the test string. That's fine but the test results need to be updated with the actual results. I've checked with Kial and I don't think the front-end does any trimming so the spaces are acceptable.

Also, due to reformatting this file since you checked in this code, you will have to merge in the current master branch to resolve the conflicts. The conflicts are really just white space.